src/test/resources/pacts/publicapi-direct-debit-connector-search-mandates-endpoint.json

```diff
@@ -0,0 +1,220 @@
+{
+  "consumer": {
+    "name": "publicapi"
+  },
+  "provider": {
+    "name": "connector"
+  },
+  "interactions": [
+    {
+      "description": "search mandates by parameters",
+      "providerStates": [
+        {
+          "name": "a gateway account with an external ids and a confirmed mandate with an email exists",
+          "params": {
+            "gateway_account_id": "2po9ycynwq8yxdgg2qwq9e9qpyrtre",
+            "mandate_id": "jkdjsvd8f78ffkwfek2a",
+            "unique_identifier": "AAAAA",
+            "charge_id": "charge97837509646393e3A",
+            "email": "rory@example.com"
+          }
+        }
+      ],
+      "request": {
+        "method": "GET",
+        "path": "/v1/api/accounts/123456/mandates",
+        "query": {
```

Comment:
Since the name of the query parameters form part of the contract between PublicApi and DDConnector I think it would be best to make this single pact test use all parameters rather than just email, what do you think? Check out `MandateSearchParams` in DDConnector for the full list and names. You'd have to change the state name, something like "a gateway account with an external id and a confirmed mandate searchable by all parameters"